src/content/faq/General.md

```diff
@@ -3,25 +3,80 @@ title: Frequently Asked Questions (FAQ)
 layout: faq
 category: General
 permalink: /faq/
-questions: Why should I use GraphQL?,Is GraphQL a database language like SQL?,Does GraphQL replace REST?,How can I learn GraphQL?,Is GraphQL frontend or backend?,Does GraphQL replace Redux or other state management libraries?,Is GraphQL only for React or JavaScript developers?,What is a GraphQL client and why would I use one?,What is the GraphQL Foundation?,How can I contribute to the GraphQL specification?
+questions: Why should I use GraphQL?,Is GraphQL a database language like SQL?,Does GraphQL replace REST?,How can I learn GraphQL?,Is GraphQL frontend or backend?,Is GraphQL only for React or JavaScript developers?,What is a GraphQL client and why would I use one?,Is GraphQL owned by Facebook?,What is the GraphQL Foundation?,How can I contribute to the GraphQL specification?
 ---
 
 ## Why should I use GraphQL?
 
+It depends on your use case, but in general, GraphQL has a few key features that stand out. For example, GraphQL enables you to:
+
+* Improve performance by [sending queries](/learn/queries/) for the exact data you need. 
+* Aggregate data from [multiple UI components](/learn/queries/#fragments) in [a single request](/learn/best-practices/#server-side-batching-caching ).
+* Create a representation of your data that feels familiar and natural ([a graph](/learn/thinking-in-graphs/#it-s-graphs-all-the-way-down)).
+* Ensure that all of your data is [statically typed](/learn/schema/) and these types inform [what queries the schema supports](/learn/introspection/).
+* [Reduce the need for breaking changes](/learn/best-practices/#versioning), but utilize a [built-in mechanism for deprecations](https://spec.graphql.org/draft/#sec-Deprecation) when you need to.
+* Examine queries and responses through community tools like [GraphiQL](https://github.com/graphql/graphiql).
```

Comment:
Something else I noticed: This is a lot of bullet points 🤪  Since we mention that there are more reasons outlined on the homepage, I'm debating cutting point 1, 2, 4 and/or 6.

 I'm thinking 6 (community tools) and 3 (graph-like data). Wdyt @IvanGoncharov / @benjie?

Reply:
I've tried to come up with a list of reasons I like GraphQL without looking at your list above, and then place them into the right order for me. It's very hard to order them!

- Every GraphQL API follows same spec: tooling (e.g. GraphiQL, codegen, etc) is very powerful and shared - worth investing in; learn once, apply everywhere
- GraphQL client synchronises data across my interface with virtually no effort
- Data's naturally in the right shape for the client, no/minimal post-processing required
- Strong typing
- Autogenerated client code (graphql-code-gen, relay compiler, etc)
- Data requirements co-located with data consumers (i.e. fragments)
- No API versioning (@deprecated, etc)
- Editor integration
- Linting integration

Reply:
I think the shared precise specification is probably the number 1 biggest reason to use GraphQL because so much of the rest of GraphQL's benefits flow from this one thing. You don't have to build your own tooling, everyone can use the same tooling!

I agree with dropping 1, I'd drop the batching from 2, I think 4 should be kept, re: point 6 tooling is incredible, but maybe it could be rewritten to something like "Incredibly rich and powerful tooling ecosystem, with GUIs (GraphiQL, Altair, GraphQL Playground, ...), Editor Integration (VSCode, Vim, ...) code generation (gql-codegen, Relay, Apollo, ...), linting, analytics (Apollo Optics), and much more". I suspect removing the parenthesis and just linking the entire point to https://graphql.org/code/ would make sense.